docs/Getting Started/Installation/index.md

```diff
@@ -189,3 +189,84 @@ from within the JavaEE directory and a JAR should be generated in the build/libs
 Glassfish 5.0.0 only works on JDK 8 and lower.
 !!!
 !@
+
+@![javascript]
+The SDL JavaScript Suite comes with prebuilt bundles of the library. There is a vanilla JavaScript distribution of the library as well as one for Node.js. They are located in the lib/js/dist and lib/node/dist directories respectively.
+
+## Vanilla JavaScript Setup
+This build allows you to create apps that run on the browser. In order to have the build JS file be imported into your HTML, you'll need to run a simple web server that can serve that JS file. We will be using [Node.js](https://nodejs.org/en/) and npm for this task, but you can use any software that lets you serve HTML and JS to the browser.
+In a new directory, save your `SDL.min.js` build there, and then run the following: `npm init`. Press Enter repeatedly through the prompts to set up some default npm configuration. Then, install the `express` package by running `npm install express --save`. Express is a popular Node.js package that allows easy setup of a server. Finally, in a new `index.js` file, save the following to it:
+```js
+const express = require('express');
+const app = express();
+app.use(express.static(__dirname));
+const PORT = process.env.PORT || 3000;
+const server = app.listen(PORT, async function () {
+    console.log('Server running on port', PORT);
+});
+```
+This code will start up a server on port 3000 on localhost, and serve any files in the directory it is running in. Now that the server code is complete, create a new file named `index.html` that will contain the app logic and save it in the same directory. Make the contents of the HTML file the following:
+```html
+<html>
+    <head>
+        <script src='./SDL.min.js'></script>
+    </head>
+    <body>
+        <script>
+            console.log(SDL);
+        </script>
+    </body>
+</html>
+```
+Finally, run the server by running this in the same directory: `node index.js`. The console should print `Server running on port 3000`. Go to your browser and enter `localhost:3000` in the address bar. If you open up your browser's console on the blank page that shows up you should see the SDL library version be printed and the imported SDL object. If so, then you have successfully set up SDL in your browser! If not, then make sure your SDL build name is correct, and that the HTML file, build file, and JS server file are all in the same directory.
+
+## Node.js Setup
+This build allows you to create apps that run on your computer. [Node.js](https://nodejs.org/en/) and npm are the tools you will need to integrate this library into what will be your app.
+In a new directory, save your `SDL.min.js` build there, and then create a new file in the same directory named `index.js`. Make the contents of that file the following:
```

Comment:
Same as above, how does a dev know where to get `SDL.min.js` or how to generate it?